hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestStorageContainerManager.java

```diff
@@ -525,7 +523,7 @@ public void testScmInfo() throws Exception {
   /**
    * Test datanode heartbeat well processed with a 4-layer network topology.
    */
-  @Test(timeout = 60000)
+  @Test(timeout = 180000)
   public void testScmProcessDatanodeHeartbeat() throws Exception {
```

Comment:
Sleep lengths in `testCloseContainerCommandOnRestart` are being increased.  Why do we need to increase timeout on another test method?

Reply:
I tested it multiple times on my laptop and tests use to timeout occasionally depending on other applications on my laptop. I kept it long enough that there should be no flakiness in the tests.